test/tests/haskell-cabal/container.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
@Moghedrin can we drop this blank line so the preamble is the same style as the other files in this repo?
```bash
#!/bin/bash
set -e

# actual code here
```